elements/pl-external-grader-variables/pl-external-grader-variables.py

```diff
@@ -1,5 +1,6 @@
 import chevron
-import lxml
+import lxml.etree
+import lxml.html
```

Comment:
👀 is this an error in the stubs?

Reply:
I'll have to double check, but I actually think this is correct and the `import lxml` was working before because of some weird Python thing. Most of the imports in PL actually use `import lxml.html` already.

Reply:
Looked again, yeah this has to do with the way that `html` and `etree` are configured within `lxml`. Frankly, it's just kindof a weird package.

Reply:
Gotcha, thanks for checking!